exercises/input.js

```diff
@@ -4,34 +4,43 @@ class Input{
 	//constructor takes in a a target input element
 	//should also construct variables for the range, the pattern, and the element that will hold the error message
 	//range min and max should default to null
-	constructor( ){
-
+	constructor( testInput ){
+		this.testInput = $(testInput);
+		this.pattern = undefined;
+		this.minRange = null;
+		this.maxRange = null;
+        this.errorMessage;
 	}
 	//setRange sets the minimum and maximum range, if necessary, for the input
 	//arguments : min (a number), and max (a number)
 	//returns: nothing
 	//purpose: sets the min and max values for the object
-	setRange(  ){
-
+	setRange( min, max ){
+		this.minRange = min;
+		this.maxRange = max;
 	}
 	//getRange gets the minimum and maximum range.
 	//arguments: nothing
 	//returns: an object with a property of min, and a property of max, containing the minimum and maximum numbers
 	getRange(){
-		
+		var rangeObj = {
```

Comment:
could have also returned this object directly, though I never fault making new variables at this stage.